build.gradle.kts

```diff
@@ -33,6 +33,10 @@ release {
   defaultVersionStrategy = Strategies.getSNAPSHOT()
 }
 
+nebulaRelease {
+  addReleaseBranchPattern("""v\d+\.\d+\.x""")
```

Comment:
Just want to confirm that patch release branches will still end in `.x`, even though though we will be releasing a specific patch version?

Reply:
Yup - subsequent patches are released from the same branch (off of the main branch tag)